frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx

```diff
@@ -302,6 +317,57 @@ const ErrorInstance: React.FC<Props> = ({ errorGroup }) => {
 									return
 								}
 
+								const queryParams: LogsSearchParam[] = []
+								if (errorObject.source) {
+									queryParams.push({
+										key: 'host.name',
+										operator: DEFAULT_LOGS_OPERATOR,
+										value: errorObject.source,
+										offsetStart: 1,
+									})
+								}
+								if (errorObject.session?.secure_id) {
+									queryParams.push({
+										key: 'secure_session_id',
+										operator: DEFAULT_LOGS_OPERATOR,
+										value: errorObject.session?.secure_id,
+										offsetStart: 2,
+									})
+								}
+								if (errorObject.trace_id) {
+									queryParams.push({
+										key: 'trace_id',
+										operator: DEFAULT_LOGS_OPERATOR,
+										value: errorObject.trace_id,
+										offsetStart: 3,
```

Comment:
Does `offsetStart` need to be adjusted based on whether the conditional actually applies? i.e. if the two above conditionals (`errorObject.source` and `errorObject.session?.secure_id`) return false, should `offsetStart` be 1 here?

Reply:
cc @ccschmitz - thoughts on the above question?
I saw this comment and I'm not sure if we're using `offsetStart`:
https://github.com/highlight/highlight/blob/d099fff437f8772f380fa2ca29272a386418487b/frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx#L385

Reply:
good catch - just updated that to increment `offsetStart` in a way that will keep them sequential.

Reply:
`offsetStart` is used to specify the index the term starts at in the query string, not the index in the array of query terms. Sorry, this logic is kind of confusing, but it's used with the cursor's index to determine what the active query term is, but the active query term isn't relevant when you are clicking in the logs table to append a filter because the input loses focus. That's why I left that comment that it wasn't used and just made it `0`.

I don't think `offsetStart` matters in the context of this PR for the same reasons and it's going to be [recalculated on each render](https://github.com/highlight/highlight/blob/e69897a25356eabe1fcd62f68c992e08e4e41a7e/frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx#L127-L129). I'd say just set it to `0` here. Let me know if that makes sense!